src/components/AuthPanel.tsx

```diff
@@ -0,0 +1,143 @@
+        const fetchTest = async () => {
+            try {
+                const res = await jwtTest()
+                console.log("res.data", res.data);
+                setCanSeeTest(true);
+            } catch (e) {
+                console.log("e", e);
+            }
+        };
+
+        fetchTest();
+    }, []);
+
+    return (
+        <div>
+            <CssBaseline/>
+
+            <Box
+                sx={{
+                    marginTop: 8,
+                    display: "flex",
+                    flexDirection: "column",
+                    alignItems: "center",
+                }}
+            >
+                <Avatar sx={{m: 1, bgcolor: "secondary.main"}}>
+                    <LockOutlinedIcon/>
+                </Avatar>
+                <Typography component="h1" variant="h5">
+                    {ACTION_MESSAGE}
+                </Typography>
+                <Box component="form" onSubmit={handleSubmit} noValidate sx={{mt: 1}}>
+
+                    <TextFieldMui autoComplete="email" label="Email Address" name="email" required size="medium"/>
+                    <TextFieldMui autoComplete="current-password" label="Password" name="password" type="password"
+                                  required size="medium"/>
+
+                    {/*<FormControlLabel*/}
+                    {/*  control={<Checkbox value="remember" color="primary" />}*/}
+                    {/*  label="Remember me"*/}
+                    {/*/>*/}
+
+                    <ButtonMui
+                        fullWidth
+                        sx={{mt: 3, mb: 2}}
+                        text={ACTION_MESSAGE}
+                        type="submit"
+                        variant="contained"
+                    />
+
+                    <Grid container>
+                        {/*<Grid item xs>*/}
+                        {/*  <Link href="#" variant="body2">*/}
+                        {/*    Forgot password?*/}
+                        {/*  </Link>*/}
+                        {/*</Grid>*/}
+                        <Grid item sx={{mb: 2}}>
+                            <ButtonMui
+                                onClick={handleToggleAuth}
+                                sx={{textTransform: "none"}}
+                                text={
+                                    isLogin
+                                        ? "Don't have an account? Sign Up"
+                                        : "Already a member? Sign In"
+                                }
+                            />
+                        </Grid>
+                    </Grid>
+                    <Divider/>
+                    {/* TODO: Use google button / render */}
+                    <ButtonMui
+                        href="api/auth/google/login"
+                        text="Sign up / Login with Google"
+                    />
+                </Box>
+            </Box>
+            <Copyright/>
+
+            <div>cat content: {canSeeTest.toString()}</div>
```

Comment:
Is CanSeeTest still necessary?

If yes, maybe we can add a build config that this will only show during dev but not during production

Reply:
Removed